lib/src/utils/builders/attachment_builder.dart

```diff
@@ -60,9 +60,11 @@ class AttachmentBuilder {
 
   /// Returns attachment encoded in Data URI scheme format
   /// See: https://discord.com/developers/docs/reference#image-data
-  String getBase64() {
+  String getBase64({String? defaultFormat = 'png'}) {
+    defaultFormat ??= 'png';
```

Comment:
No need for this, the parameter can be non nullable with a default value

Reply:
better to remove `?` from type. Makes no sense if default parameter is present

Reply:
Yeah, sorry, I'm not paying much attention today 😅